frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,108 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  addCapacityLbl,
+  addCapacityBtn,
+  ocsOp,
+  storageClusterView,
+  storageClusterRow,
+} from '../views/add-capacity.view';
+
+const NAMESPACE = 'openshift-storage';
+const KIND = 'storagecluster';
+const EXPANDWAIT = 150000;
+const storageCluster = JSON.parse(
+  execSync(`kubectl get -o json -n ${NAMESPACE} ${KIND}`).toString(),
+);
+
+let clusterJSON;
+let previousCnt;
+let previousPods;
+let updatedClusterJSON;
+let updatedCnt;
+let updatedPods;
+
+describe('Check add capacity functionality for ocs service', () => {
+  beforeAll(async () => {
+    // check whether the cluster is present or not, and check for its ready state
+    // if not present exit and console that no cluster is present, will be send in next PR
+
+    const { name } = storageCluster.items[0].metadata;
+    clusterJSON = JSON.parse(
+      execSync(`kubectl get -o json -n ${NAMESPACE} ${KIND} ${name}`).toString(),
+    );
+    previousCnt = _.get(clusterJSON, 'spec.storageDeviceSets[0].count', undefined);
+    const uid = _.get(clusterJSON, 'metadata.uid', undefined).toString();
```

Comment:
```suggestion
    const uid = _.get(clusterJSON, 'metadata.uid').toString();
```
lodash return undefined by default